src/Page/Guide/View.elm

```diff
@@ -149,7 +148,7 @@ viewRelatedGuideTeasers language guideTitleList allGuidesSlugTitleList =
             , ul [ css [ listStyleNone ] ]
                 (List.map
                     (\item ->
-                        li [] [ a [ href item.slug ] [ text (titleFromLanguage language item) ] ]
+                        li [] [ a [ href (Route.toString (Guide item.slug)) ] [ text (titleFromLanguage language item) ] ]
```

Comment:
now I'm looking at this I'm surprised it was working in the first place

Reply:
Yes - this also confused me. There is another pattern used with the teasers that include images - so we do need to check that's all resolving correctly.